src/samples/DepthMap.hpp

```diff
@@ -239,8 +239,8 @@ struct DepthMap
 	point_cloud.reserve(distances.size());
 	
 	// precompute local transformations
-	std::vector< Eigen::Transform<typename T::Scalar,3,Eigen::Affine> > rows2column;
-	std::vector< Eigen::Transform<typename T::Scalar,3,Eigen::Affine> > columns2pointcloud;
+	std::vector< Eigen::Transform<typename T::Scalar,3,Eigen::Affine>, Eigen::aligned_allocator< Eigen::Transform<typename T::Scalar,3,Eigen::Affine> > > rows2column;
```

Comment:
Eigen types with alignment constraints should never be used on these types, as it breaks on any form of reflection
(e.g. typelib and RTT).

This is the reason for the base::* typedefs. Create base:: typedefs for the relevant types (transform and friends).